app/templates/awards_filters.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
```

Comment:
I suggest to extend base.html, if possible.

Reply:
It is not possible due to integration. The filter widget stays independently on the page and by extending the base would result to have a navigation menu attached.

Reply:
Ok, I see.